pkg/egresspolicy/egresspolicy.go

```diff
@@ -0,0 +1,149 @@
+//  Copyright 2021 Authors of Cilium
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package egresspolicy
+
+import (
+	"fmt"
+	"net"
+
+	k8sConst "github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
+	v2alpha1 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
+	k8sLabels "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/labels"
+	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/cilium/cilium/pkg/policy"
+	"github.com/cilium/cilium/pkg/policy/api"
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// EgressPolicyConfig is the internal representation of Cilium Egress NAT Policy.
+type EgressPolicyConfig struct {
```

Comment:
When loading this into my editor, it warns that this name"stutters" because it's part of package `egresspolicy` and would be referred to from outside modules as `egresspolicy.EgressPolicyConfig`. I think a quick rename to `Config` would be nice; shouldn't affect any functionality.

Reply:
fixed. thx!